<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_cliente</class>
 <widget class="QDialog" name="dialog_cliente">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>771</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Consultar</string>
     </attribute>
     <widget class="QTableView" name="tb_clientes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CONSULTA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_buscar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buscar">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por cliente</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_apellido">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Observacion</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cedula">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por fecha</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modificar</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>581</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Paciente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Observaciones</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Observaciones</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="comboBox_3">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>1</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_guardar_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>360</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MODIFICAR CONSULTA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Ingresar</string>
     </attribute>
     <widget class="QPushButton" name="btn_guardar">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>360</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INGRESAR CONSULTA</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>581</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Paciente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Observaciones</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Observaciones</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>1</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_guardar_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Eliminar</string>
     </attribute>
     <widget class="QPushButton" name="btn_eliminar_selec">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Seleccion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eliminar_todo">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Todo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR CONSULTA</string>
      </property>
     </widget>
     <widget class="QTableView" name="tb_clientes_eliminar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
